kubectl-rook-ceph.sh

```diff
@@ -328,8 +330,18 @@ function run_main_command () {
 # MAIN: PARSE MAIN ARGS AND CALL MAIN COMMAND HANDLER
 ####################################################################################################
 
-# Default values for flag-controlled settings.
-NAMESPACE='rook-ceph' # namespace of the cluster
+# Default values
+NAMESPACE=$(printenv ROOK_CLUSTER_NAMESPACE ) # namespace of the cluster
+OPERATOR_NAMESPACE=$(printenv ROOK_OPERATOR_NAMESPACE ) # operator namespace
+CEPH_CONF_PATH="/var/lib/rook/$NAMESPACE/$NAMESPACE.config" # path of ceph config
```

Comment:
Need to move this down a few lines until after the namespace vars are finally determined?

Reply:
Seems like lines 336-344 should move down around 377 so these defaults will be set only after the command line args are evaluated?